### ISSUE TITLE
hand_tracking_samples on linux?

### ISSUE THREAD

**Issue by RoboEvangelist** [IntelRealSense/hand_tracking_samples#5]
Folks,

I noticed the samples are for Visual Studio only, and not for Linux.
Any current work to solve this problem?

**Comment by ddiakopoulos** [comment]
Hi @RoboEvangelist - you might have simply missed the Makefile in the root directory. Support and instructions are limited, but this codebase and its dependencies were recently tested on Ubuntu 16.10

**Comment by RoboEvangelist** [comment]
I get this error when I do "make" on my terminal on Ubuntu 16.04:

```
mkdir -p bin
mkdir -p obj
clang++ dataset-exporter/dataexporter.cpp -std=c++14 -Wno-narrowing -Iinclude -fdelayed-template-parsing -Ofast -march=native -c -o obj/dataexporter.o
make: clang++: Command not found
Makefile:66: recipe for target 'obj/dataexporter.o' failed
make: *** [obj/dataexporter.o] Error 127
```

**Comment by lucieb** [comment]
Hello @RoboEvangelist - it looks like the clang compiler is not installed on your machine. Open a terminal and install the clang package using the command:
```
sudo apt-get install clang
```
It will prompt you for your password and for the authorization to install the packages, simply follow the instructions. 

Before running the Makefile again, make sure to checkout the latest version of the code, I just made a fix in the Makefile.

Once the clang installation is done and you have the latest code, clean your directory and run the makefile again.
```
make clean
make
```

I just tested it on Ubuntu 16.04.
